bubuku/cli.py

```diff
@@ -103,11 +102,15 @@ def rolling_restart_broker(image_tag: str, instance_type: str, scalyr_key: str,
 @click.option('--bin-packing', is_flag=True, help="Use bean packing approach instead of one way processing")
 @click.option('--parallelism', type=click.INT, default=1, show_default=True,
               help="Amount of partitions to move in a single rebalance step")
-
 @click.option('--throttle', type=click.INT, default=100000000, help="Upper bound on bandwidth (in bytes/sec) used for "
                                                                     "rebalance")
 def rebalance_partitions(broker: str, empty_brokers: str, exclude_topics: str, parallelism: int, bin_packing: bool,
                          throttle: int):
+
+    if throttle and throttle < get_max_bytes_in():
+        print('Throttle value must be set above the max BytesIn for the replication to progress. '
+              'The current max BytesIn is {}'.format(get_max_bytes_in()))
+        return
```

Comment:
can you please do exit(1) so calling process can have some info about unsuccessful? result